torchgeo/trainers/base.py

```diff
@@ -41,7 +42,9 @@ def configure_metrics(self) -> None:
     def configure_models(self) -> None:
         """Initialize the model."""
 
-    def configure_optimizers(self) -> dict[str, Any]:
+    def configure_optimizers(
+        self,
+    ) -> "lightning.pytorch.utilities.types.OptimizerLRSchedulerConfig":
```

Comment:
We can remove these quotes once our min dep is >= 2.0.9. Didn't feel like bumping just for this reason.